README.md

```diff
@@ -1,2 +1,174 @@
+
+### Automated scripts 
+
+Automated scripts checks triggered by no unit tests SDL build jobs. 
+Each job contains a description of ATF test sets that it includes. 
+
+### Automated smoke tests 
+
+Contains basic checks from [smoke_tests.txt](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/master/test_sets/smoke_tests.txt)
+
+A test set executed for SDL build in 3 policy flows: 
+
+ - [![bage][develop automated smoke proprietary bage]][develop automated smoke proprietary]
+ - [![bage][develop automated smoke external proprietary bage]][develop automated smoke external proprietary]
+ - [![bage][develop automated smoke http bage]][develop automated smoke http]
+ 
+
+### Automated regression
+
+#### Automated policy regression 
+
+Check pull regression specific for each policy flow.
+
+A test set executed for SDL build in 3 policy flows: 
+ - [![bage][proprietary atf policy bage]][proprietary atf policy ]
+ - [![bage][external proprietary atf policy bage]][external proprietary atf policy ]
+ - [![bage][http atf policy bage]][http atf policy]
+
+#### Automated RC regression 
+
+#### Various features regression
+Contains all ATF scripts for all featured available in the development.
+For parallel execution, checks are split to multiple jobs with make name template **Develop_TCP_ATF_VF{X}_{P,H,E}**. **X** is the number.
+
+Full list of regression ATF jobs available on : https://opensdl-jenkins.prjdmz.luxoft.com/view/ATF_Regression_on_TCP/ 
+
+## PR to develop checks
+
+Pull request jobs are available in view: https://opensdl-jenkins.prjdmz.luxoft.com/view/PR_checks/ 
+
+PR checks : 
+ - code style
+ - build with unit tests
+ - [smoke_tests.txt](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/master/test_sets/smoke_tests.txt) trigger by build with unit tests.
+
+## Weekly checks 
+
+List of develop jobs executed [weekly](https://opensdl-jenkins.prjdmz.luxoft.com/view/Develop_weekly/)
+
+### Unit tests coverage 
+
+Unit tests coverage check weekly in the job: [![bage][unit test coverage bage]][unit test coverage]
+
+### Full ATF regression
+
+On Saturday, SDL CI performs a full regression check on these following transports: TCP, WebSockets, SecureWebSockets, and all 3 policy flow. 
+Weekly builds are available on the view https://opensdl-jenkins.prjdmz.luxoft.com/view/Develop_weekly/ 
+
+Each SDL build without unit tests triggers full ATF regression. 
+List of triggered jobs available in each build job as Downstream projects, example:
+https://opensdl-jenkins.prjdmz.luxoft.com/view/all/job/Develop_SDL_NoUT_E_BWSS_OFF/ 
+
+Full weekly status is available on https://opensdl-jenkins.prjdmz.luxoft.com/view/Develop_weekly_status/ 
+
+## Feature checks:
+
+For each feature before merging to develop should be created a list of jobs similar to develop to check that feature will no introduce a regression. 
+There is a special job in CI [Feature job create]() that will create a list of jobs and a separate view for the feature.
+
```

Comment:
Let's put a link to https://opensdl-jenkins.prjdmz.luxoft.com/view/Jenkins_Utils/ and some description for all three jobs